projects/fsastorefrontlib/src/cms-components/my-account/claim/claim-details/claim-details.component.html

```diff
@@ -104,14 +104,6 @@ <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
               {{ subclaim.status }}
             </div>
           </li>
-          <li class="accordion-list-item row">
-            <div class="col-6 col-sm-5 item-label">
-              {{ 'claim.subclaim.statusDescription' | cxTranslate }}
-            </div>
-            <div class="col-6 col-sm-5 value notice">
-              <!-- {{ display statusDescription }} -->
```

Comment:
Not related to this task, I removed `statusDescription` as we agreed.

Reply:
But then shouldn't the wrapping div be deleted as well?

Reply:
I don't understood clearly, I removed whole `li` element `accordion-list-item row`.

Reply:
Sorry, my bad, I overlooked it.